pkg/controller/factory/factory.go

```diff
@@ -68,6 +68,11 @@ type PostStartHook func(ctx context.Context, syncContext SyncContext) error
 // triggers.
 type ObjectQueueKeyFunc func(runtime.Object) string
```

Comment:
do we need this?

Reply:
it is still needed as the parameter of WithInformersQueueKeyFunc